lib/email_tracking_retriever.py

```diff
@@ -67,9 +67,8 @@ def get_trackings(self) -> Dict[str, Tracking]:
     # Incomplete tracking information from emails with handled errors.
     incomplete_trackings = []
 
-    self.driver = self.driver_creator.new()
     try:
-      log_in_if_necessary(self.driver, self.config)
+      self.driver = self.log_in_if_necessary()
```

Comment:
this probably doesn't work in the bby world, did you test it? 
1. this would need to be outside the try / finally block
2. we call self.driver.quit() later, so it needs to be initialized in that case

Reply:
I did test it and it worked, just want to confirm your saying to place this line above the try block and we do not need to call driver.quit() since its being done elsewhere?